stellarphot/settings/models.py

```diff
@@ -221,9 +249,10 @@ class Camera(BaseModelWithTableRep):
     model_config = MODEL_DEFAULT_CONFIGURATION
 
     name: Annotated[
-        str,
+        NonEmptyStr,
```

Comment:
Took me a bit, but I see this is now using that NonEmptyStr validator you set up above...